### DISCUSSION TITLE
v1.4-alpha.2

### DISCUSSION THREAD

**Discussion by arekf** [yattee/yattee#93] [Announcements]
**You are encouraged to leave your feedback in the release discussion.**
Ideas on player controls are welcome but please do not report bugs/improvements requests focusing on the current controls which will be rebuilt in the near future to meet needs of each supported platform.

## What's Changed
* tvOS build is now available
* Added new quality settings
* Fix drawing player on some devices
* Fix closing fullscreen on closing player
* Includes fixes from 1.3.3 release

**Full Changelog**: https://github.com/yattee/yattee/compare/v1.4-alpha.1...v1.4-alpha.2

<hr /><em>This discussion was created from the release <a href='https://github.com/yattee/yattee/releases/tag/v1.4-alpha.2'>v1.4-alpha.2</a>.</em>

**Comment by bachig26** [comment]
is there any plan to add a refresh button, like nearby the account icon?

**Reply by arekf** [reply]
Subscriptions/Trending/Popular has pull to refresh or command+r. It is rather complex to add to other views like Favorites. More for the future than for now.

**Comment by bachig26** [comment]
**Idea 1:** Introducing the _**Jump to Next video/Jump to Previous video**_ button in the player; this will be useful when the playlists or queues are played.

**Reply by arekf** [reply]
Created an issue for this as is planned https://github.com/yattee/yattee/issues/106

**Comment by LL4ur3nz** [comment]
If i rotate to Landscape i just get a black screen, how to fix it?

**Reply by arekf** [reply]
It's a bug in current build that is happening on some devices, I will work on resolving it in future releases. Can you report what device are you using?

**Reply by LL4ur3nz** [reply]
Yeah 12 pro Max, in the last build 1.3.2 i couldn't even see anything also nit in portrait... so you fixed something i think :)

**Reply by arekf** [reply]
You're running into #78. I was not able to fully fix it yet but it's on my list for next build so stay tuned.

**Reply by arekf** [reply]
A temporary workaround to open fullscreen properly on Max device is to disable "lock portrait mode" in browsing options and open video while browsing landscape.

**Comment by c1sxr0bzn** [comment]
Really like the fact that I can now automatically watch 1080p and up on my TV without manually switching, however, does the player get changed to the old look? I really did like that flush TV look.

**Reply by arekf** [reply]
Current alpha player controls are temporary and will be improved for each platform in the future, including Apple TV. It's not gonna be exactly the same look as the old one, but I want to deliver more info and customization than before.
